CHANGES/1231.bugfix

```diff
@@ -0,0 +1 @@
+Tought registry to accept docker schema2 sub-manifest types in oci index.
```

Comment:
```suggestion
Taught the Pulp Registry to accept docker schema2 sub-manifest types in OCI index.
```

Reply:
I was thinking this word was looking weird to me...